server/game/cards/02.1-NTNR/MakeEmSweat.js

```diff
@@ -0,0 +1,35 @@
+const ActionCard = require('../../actioncard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class MakeEmSweat extends ActionCard {
+    setupCardAbilities(ability) {
+        this.action({
+            playType: 'shootout',
+            cost: ability.costs.boot(card =>
+                card.getType() === 'dude' &&
+                card.isParticipating() && card.controller === this.controller &&
+                    !card.booted),
+            target: {
+                cardCondition: {location: 'play area', condition: card => card.isParticipating()},
+                cardType: 'dude'
+            },
+            handler: context => {
+                this.applyAbilityEffect(context.ability, ability => ({
+                    match: context.target,
+                    effect: ability.effects.modifyBullets(-context.costs.boot.bullets)
+                }));
+                if(context.target.bullets <= 0 && !context.target.booted) {
+                    this.game.resolveGameAction(GameActions.bootCard({ card: context.target }), context).thenExecute(() => {
+                        this.game.addMessage('{0} uses {1} to boot {2} and reduce {3}\'s bullets by {4} and boot {3}', context.player, this, context.costs.boot, context.target, context.costs.boot.bullets);
```

Comment:
I would change the message to make it clear the first booting is a cost
```suggestion
                        this.game.addMessage('{0} uses {1} and boots {2} to reduce {3}\'s bullets by {4} and boot {3}', context.player, this, context.costs.boot, context.target, context.costs.boot.bullets);
```